financials/income_log.csv

```diff
@@ -18,4 +18,113 @@ income_id,date,type,source,currency,amount,exchange_rate,USD_equivalent,to_addre
+105,20181127,donation,unknown,BTC,0.002,3820.72,7.64,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj75,05f02afeff6d6c4d1402571e005049d87735a6ac07a85216d440313a9c8f528f,,
+106,20181128,donation,unknown,BTC,0.05,4257.42,212.87,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj74,52b196fe7c76e96055592ca4662e73a746104f59dc8134f1a139da431b7bd36f,,
+107,20181129,donation,unknown,BTC,0.10001797,4278.85,427.96,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj73,1cd28d315728c10a71d4327b929e408a2ac5cd17189cee1eeab28676a5bd86f0,,
+108,20181201,donation,unknown,BTC,0.12777,4214.67,538.51,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj72,02af10853ccabb961143fa0187689c9bfab0d69a9dfcb81283af5ee3eab0620e,,
+109,20181203,donation,unknown,BTC,1,3894.13,3894.13,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj71,a81b3cbaba43fa70b3f5be8bbc2d16d3dee06d3617faa7a6dab73c210b0edf2d,,
+110,20181205,donation,unknown,BTC,0.0015,3753.99,5.63,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj70,45052bffdd51c68c50e5d9dff46aba7a50d922cb7efc3807c5edcd2c2aec8807,,
+111,20181221,donation,unknown,BTC,0.03,3896.54,116.90,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj69,76ea404a121ff508ba3d5881046342c72cd5951057e283acc8646e850bf90d6a,,
+112,20190118,donation,unknown,BTC,0.03190333,3657.84,116.70,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj68,d88b2449544e414d411e0c6f9ceb22cad5d7260b41ffc91aafe4bcce35308115,,
+113,20190206,donation,unknown,BTC,0.005,3413.77,17.07,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj67,198ec6871084c894e88336872a5d7e035d4034e917858cf7fae48a281c66080d,,
+114,20190209,donation,unknown,BTC,0.00136946,3671.20,5.03,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj66,61a4a4d01de89eb109ae68fbd4be2b36bb76a773f4fc8dd3b43cfa3972fea4d3,,
+115,20190210,donation,unknown,BTC,3.5,3690.19,12915.67,bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj65,ce409d7a4e193edfba1f13fe2698ce104956733d1a286f9935290977abce0312,,
+116,20181202,donation,unknown,ZEC,0.2,79.16,15.83,t1QGYYXan3HHEuiPEfccKnUuWEP4CsVvPA24,451aa2c711d314d0afc7be9d5944cfb3dc98fa83c8aa51aca2f11c9c75f9a95e,,
+117,20181203,donation,unknown,ZEC,0.2,70.8,14.16,t1QGYYXan3HHEuiPEfccKnUuWEP4CsVvPA21,51891a983f1d640e879f1cc5c3baa822e34f9f0228dd1b7916fa376bcb9a6fd6,,
+118,20181203,donation,unknown,ZEC,0.2,70.8,14.16,t1QGYYXan3HHEuiPEfccKnUuWEP4CsVvPA22,b10ba22504ac41da06e7c6be0334a3f8e43c0b1b1774844c80b752657da287f2,,
+119,20181203,donation,unknown,ZEC,0.2,70.8,14.16,t1QGYYXan3HHEuiPEfccKnUuWEP4CsVvPA23,9d9a232a2ed10077d84b28dc6a9ef11483fe6aabf17fb64ab9499989886c183d,,
+120,20181204,donation,unknown,ZEC,0.2,74.69,14.94,t1QGYYXan3HHEuiPEfccKnUuWEP4CsVvPA20,8e9814f301a0c9e9626a391046a88566f64726ee85d356f6bdcb4d95395b17de,,

```

Comment:
Where are these addresses coming from? I only see t1QGYYXan3HHEuiPEfccKnUuWEP4CsVvPA5 being a valid ZEC account.

Reply:
Ah, that's stupid spreadsheet behaviour done by mistake, it should always be the same address. Great catch @kargakis - just pushed a commit that should fix this.